docs/source/configuration/skip_formatting.rst

```diff
@@ -37,7 +37,7 @@ Both options are configurable using configuration file (:ref:`config-file`).
         "NormalizeSeparators:skip_documentation=False"
     ]
 
-.. _skip return_values:
+.. _skip return values:
```

Comment:
Small typo that resulted in not working url.

Reply:
Where is the typo? :D

Reply:
Underscore instead of space - looks like github doesn't display it correctly